odoo/addons/base/models/ir_actions.py

```diff
@@ -49,6 +49,7 @@ class IrActions(models.Model):
                                      ('report', 'Report')],
                                     required=True, default='action')
     binding_view_types = fields.Char(default='list,form')
+    display_in_controlpanel = fields.Boolean(string='Display in Controlpanel', default=False)
```

Comment:
Control panel is in two words.
This also needs an `help` attribute also to explain what will be the effect

Reply:
Hello @mat
Changes are done.
And for contextual action, I have done as per the spec.
do you like to change it ?
Thanks